src/verify/VerificationCodeInput.test.tsx

```diff
@@ -105,28 +104,4 @@ describe('VerificationCodeInputWrapper', () => {
     })
     expect(mockSetSmsCode).toHaveBeenCalledWith('456789')
   })
-
-  it('shows paste button if clipboard has content matching code length', async () => {
```

Comment:
paste button removed in the latest designs. OS keyboards now have the prompt for pasting in codes